src/systemtest/java/com/ericsson/ei/systemtest/artifactflow/ArtifactFlowSteps.java

```diff
@@ -36,7 +32,27 @@ public void configurations_are_provided() {
         config.initRemRemConfig();
     }
 
-    @Given("^a jenkins job '\\\"([^\\\"]*)\\\"' from '\"([^\"]*)\"' is created with parameters: (.*)$")
+    @Given("^subscription object \"([^\"]*)\" is created which will trigger \"([^\"]*)\"$")
+    public void subscription_is_created(String subscriptionName, String nameOfTriggeredJob) throws Throwable {
```

Comment:
hasParameters needs to be added in this file as well :)

Reply:
Done